### PR TITLE
Create cmake-single-platform.yml

### PR DESCRIPTION
**Pull request by purrpgoddpeebles**

### CODE REVIEW

File: .github/workflows/cmake-single-platform.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Configure CMake
+      # Configure CMake in a 'build' subdirectory. `CMAKE_BUILD_TYPE` is only required if you are using a single-configuration generator such as make.
+      # See https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html?highlight=cmake_build_type
+      run: cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}}
```
Comment (Jamiras) [inline .github/workflows/cmake-single-platform.yml:28]: This project doesn't use cmake

**Review by Jamiras — COMMENTED** [review]
What is the purpose of this PR? Please include a description.

It looks like you're trying to build RAlibretro natively on linux, but doing so using a compile chain that it doesn't support.

And assuming you do manage to get it to build, it won't run correctly as it will try to load the integration DLL, which is Windows only.

**Comment by purrpgoddpeebles** [comment]
Yeah I messed up and didn't know what I was doing I'm trying to learn how
to even use GitHub I'm using an iPhone A12 and I'm running iOS 17.3 beta
with developer mode on I'm just trying to get an emulator from GitHub that
I can run on my iPhone it's been a difficult process as a lot of them use
Xcode and I have no clue how to use any of it do you have an IPA or iPhone
version of RA libretro that you can share with me one that I can easily
install I would greatly appreciate it

**Comment by Jamiras** [comment]
Because of the DLL limitation, RAlibretro effectively only works on Windows. For RetroAchievements on iPhone, look at RetroArch.

**Pull request closed without merging**